### ISSUE TITLE
optimize encoder performance

### ISSUE THREAD

**Issue by jeff-regier** [prob-ml/bliss#727]
- [x] explore whether pretraining with crowded field helps
- [x] tune the optimizer scheduler (try OneCycleLR and higher rates up front)
- [x] Consider different batch and images sizes and various amounts of padding
- [x] consider different anchor sizes
- [x] ~~consider using a different NN head (but same backbone) for GalSim parameters~~
- [x] ~~consider replacing/augmenting batchnorm with group-norm or instance-norm~~
- [ ] ~evaluate whether/which log, sigmoid, coadd, and z-score transformations of images improve encoder performance~ (tracked by new issue #829)
~-[ ] tune the encoder architecture both for accuracy and speed~ (tracked by new issue #822)
- [ ] save the best performing model in the `data/pretrained_models` directory

For all these, [Hydra multi-run](https://hydra.cc/docs/tutorials/basic/running_your_app/multi-run/) should be helpful. We'd like to keep a record or what did/didn't work well.

For comparative benchmarks, runs should be repeated 5 to 10 times per dataset with different seeds to quantify variation due to randomness in the weight initialization. We should also repeat the experiments with 2 or 3 (synthetic) datasets to assess whether our results hold up across datasets.

**Comment by zhixiangteoh** [comment]
WIP - https://docs.google.com/spreadsheets/d/1JTkKxC6ApoJUPlYAn51TnSZ6BG0Neykl7-1bSTlyiR8

**Comment by jeff-regier** [comment]
The remaining tasks are tracked by other issues
